src/services/UserService.ts

```diff
@@ -0,0 +1,91 @@
+import axios from "axios";
+import CreateAccountRequest from "models/CreateAccountRequest";
+import CreateAccountResponse from "models/CreateAccountResponse";
+import HasAccountRequest from "models/HasAccountRequest";
+import HasAccountResponse from "models/HasAccountResponse";
+import SignInRequest from "models/SignInRequest";
+import SignInResponse from "models/SignInResponse";
+import IUser from "models/IUser";
+
+//if I end up implementing a bunch of similar stuff I can make this a class
+// TODO: Hardcode routes into a url object or somethin similar - domain will be controlled by an env var
+
+const customConfig = {
+    headers:{
+      'Content-Type': 'application/json'
+    }
+};
+
+
+  export async function CreateAccount(request: CreateAccountRequest): Promise<CreateAccountResponse> {
+    try{
+        const response = await axios.post<CreateAccountResponse>("https://localhost:7247/api/Auth/CreateAccount",JSON.stringify(request), customConfig)
+        const result = response.data;
+        return result;
+    }
+    catch(err){
+        //Test this case
```

Comment:
Make a ticket to deal with 4xx 5xx Reponses in general probably build some kind of httpproxy that implements a Success and Failure